debian/assets/carlos_ctl/o19etl.py

```
@@ -0,0 +1,2016 @@
+    appended = appended or {}
+    src_tables = {r[0] for r in plain_query(
+        "SELECT TABLE_NAME FROM information_schema.TABLES WHERE "
+        "TABLE_SCHEMA = '{0}'".format(src_schema))}
+    for table, entry in sorted(o19map_schema.TABLES.items()):
+        if table not in src_tables:
+            continue
+        if entry.get("fk_remap"):
+            # the copy dropped remaps whose parent is absent from this
+            # dump, so no id map exists for them: the parity join must
+            # drop them too or it references a table that was never made
+            entry = dict(entry, fk_remap={
```

Ruff (0.16.3)

[warning] `for` loop variable `entry` overwritten by assignment target

(PLW2901)